### PR TITLE
Mosins are no longer able to be fired while holding stuff in the offhand BUT are back to 60 damage

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

takes forever to chamber a round in one might as well be worth shooting it compared to a shotgun
### Why is this change good for the game?
it's not
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
7.62 rounds deal 60 damage up from 40, mosins require offhand free to fire

# Changelog

:cl:  
tweak: 7.62 rounds now deal 60 damage up from 40
tweak: mosins now require the offhand to be free to fire
/:cl:

### CODE REVIEW

**Comment by Jarod1200** [comment]
LETS FUCKING GOOOO

**Comment by alexkar598** [comment]
2 mosin + boh?

**Review by JamieD1 — APPROVED** [review]

**Comment by SomeguyManperson** [comment]
> 2 mosin + boh?

boh exists just so people can completely ignore weight limitations on items so lmao
